plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -356,22 +366,81 @@ public void widgetSelected(SelectionEvent e) {
 		};
 	}
 
+	private String substitueVariables(String rawSourcePath) {
```

Comment:
@fbricon , sorry that I did not mention in my comment above about toPortableString() that I meant all its occurrences. There are 3 more of them, they all are applied to container full path implying the same arguments.